src/nupic/algorithms/Anomaly.hpp

```diff
@@ -112,10 +113,16 @@ namespace nupic
                        const std::vector<UInt>& predicted,
                        Real64 inputValue=0, UInt timestamp=0);
 
+//helper
+        void setLikelihood(AnomalyLikelihood newInstance) {
```

Comment:
Currently not all parameters in Likelihood can be set in Anomaly, so this helps by allowing to fine-tune and instantiate Likelihood as needed and then just plug it in here. (?)